src/views/Settings.vue

```diff
@@ -88,14 +88,18 @@ export default defineComponent({
   methods: {
     setEComStore(store: any) {
       if(this.userProfile) {
+        const productStoreId = store['detail'].value
         this.store.dispatch('user/setEcomStore', {
-          'eComStore': this.userProfile.stores.find((str: any) => str.productStoreId == store['detail'].value)
+          'productStoreId' : productStoreId ? productStoreId : {}
         })
       }
     },
     setShopifyConfig(event: any){
-      const shopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value)
-      this.store.dispatch('user/setCurrentShopifyConfig', shopifyConfig);
+      // const currentShopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value);
+      const shopifyConfigId = event.detail.value;
+      this.store.dispatch('user/setCurrentShopifyConfig', {
+        'shopifyConfig' : shopifyConfigId ? shopifyConfigId : {}
```

Comment:
Variable name should be shopifyConfigId
Why this is needed? We could just pass the shopifyConfigId